dnf5/include/dnf5/context.hpp

```diff
@@ -76,6 +77,10 @@ class Context : public libdnf5::cli::session::Session {
     std::vector<std::string> enable_plugins_patterns;
     std::vector<std::string> disable_plugins_patterns;
 
+    bool should_store_offline = false;
```

Comment:
I'd rather be for wrapping it into get / set methods instead of having a variable on public API.

Reply:
Also please attach some docstring.

Reply:
Done, thanks!